src/languages/en.ts

```diff
@@ -2031,38 +2045,49 @@ export default {
-        cashExpenseWithNoReceipt: 'dummy.violations.cashExpenseWithNoReceipt',
-        categoryOutOfPolicy: 'dummy.violations.categoryOutOfPolicy',
-        conversionSurcharge: 'dummy.violations.conversionSurcharge',
-        customUnitOutOfPolicy: 'dummy.violations.customUnitOutOfPolicy',
-        duplicatedTransaction: 'dummy.violations.duplicatedTransaction',
-        fieldRequired: 'dummy.violations.fieldRequired',
-        futureDate: 'dummy.violations.futureDate',
-        invoiceMarkup: 'dummy.violations.invoiceMarkup',
-        maxAge: 'dummy.violations.maxAge',
-        missingCategory: 'dummy.violations.missingCategory',
-        missingComment: 'dummy.violations.missingComment',
-        missingTag: 'dummy.violations.missingTag',
-        modifiedAmount: 'dummy.violations.modifiedAmount',
-        modifiedDate: 'dummy.violations.modifiedDate',
-        nonExpensiworksExpense: 'dummy.violations.nonExpensiworksExpense',
-        overAutoApprovalLimit: 'dummy.violations.overAutoApprovalLimit',
-        overCategoryLimit: 'dummy.violations.overCategoryLimit',
-        overLimit: 'dummy.violations.overLimit',
-        overLimitAttendee: 'dummy.violations.overLimitAttendee',
-        perDayLimit: 'dummy.violations.perDayLimit',
-        receiptNotSmartScanned: 'dummy.violations.receiptNotSmartScanned',
-        receiptRequired: 'dummy.violations.receiptRequired',
-        rter: 'dummy.violations.rter',
-        smartscanFailed: 'dummy.violations.smartscanFailed',
-        someTagLevelsRequired: 'dummy.violations.someTagLevelsRequired',
-        tagOutOfPolicy: 'dummy.violations.tagOutOfPolicy',
-        taxAmountChanged: 'dummy.violations.taxAmountChanged',
-        taxOutOfPolicy: 'dummy.violations.taxOutOfPolicy',
-        taxRateChanged: 'dummy.violations.taxRateChanged',
-        taxRequired: 'dummy.violations.taxRequired',
+        allTagLevelsRequired: 'All tags required',
+        autoReportedRejectedExpense: ({rejectReason, rejectedBy}: ViolationsAutoReportedRejectedExpenseParams) => `${rejectedBy} rejected this expense with the comment "${rejectReason}"`,
+        billableExpense: 'Billable no longer valid',
+        cashExpenseWithNoReceipt: ({amount}: ViolationsCashExpenseWithNoReceiptParams) => `Receipt required over ${amount}`,
+        categoryOutOfPolicy: 'Category no longer valid',
+        conversionSurcharge: ({surcharge}: ViolationsConversionSurchargeParams) => `Applied ${surcharge}% conversion surcharge`,
+        customUnitOutOfPolicy: 'Unit no longer valid',
+        duplicatedTransaction: 'Potential duplicate',
+        fieldRequired: 'Report fields are required',
+        futureDate: 'Future date not allowed',
+        invoiceMarkup: ({invoiceMarkup}: ViolationsInvoiceMarkupParams) => `Marked up by ${invoiceMarkup}%`,
+        maxAge: ({maxAge}: ViolationsMaxAgeParams) => `Date older than ${maxAge} days`,
+        missingCategory: 'Missing category',
+        missingComment: 'Missing comment',
```

Comment:
I missed this one when I shared the definitive list with you
```suggestion
        missingComment: 'Description required for selected category',
```

Reply:
done!